### PR TITLE
FakeSome and sketch

### PR DESCRIPTION
**Pull request by caroribada**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -26,3 +26,22 @@ function fakeMap(arr, fn) {
   }
   return aux;
 }
+
+var fakeSome = function(array, condition) { //declaro funcion fakeSome
```
Comment (eebalcarcel) [inline app.js]: Change the declaration to explicit function declaration, like this: **function _fakeSome_(array, condition){ }**

**Comment by eebalcarcel** [comment]
Remove all console.logs and you should always use **const** for all the references except when you know that variable is going to be reassigned, then you should use **let**

File: app.js [hunk 2]
```diff
@@ -26,3 +26,22 @@ function fakeMap(arr, fn) {
...
+      var result = condition(value); //A partir de la cuenta condition le estoy pasando la variable que tomo en value que es element % 2 === 0 - result va a ser true or false
+      console.log(value); //imprimir el valor de la lista
+      if (result === true) { //si condition es par devuelve true
+          return true; 
+      }
+  }
+  return false; //si el resultado es impar
+}
+
+var arrayDePrueba = [0, 0, 5, 3]; //declara e inicializa el array
```
Comment (eebalcarcel) [inline app.js]: Remove everything that's used for testing

File: styles/sketch.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    border-top-color: rgb(255, 255, 255);
+    border-top-style: none;
+    border-top-width: 0px;
+    color: rgb(255, 255, 255);
+    font-family: Arial, Helvetica, sans-serif;
+    display: inline-block;
+    font-size: 23.111px;
+    font-weight: 400;
+    line-height: 28.8888px;
+    padding-bottom: 0px;
```
Comment (eebalcarcel) [inline styles/sketch.css]: For padding and margin, in this case, you can use the shorthand, like this: **padding: 0 4px 0 40px;**

Comment (caroribada) [reply styles/sketch.css]: done

**Pull request closed without merging**